src/structures/Chat.js

```diff
@@ -17,6 +17,8 @@ class Chat extends Base {
         this.id = data.id;
 
         this.name = data.formattedTitle;
+        this.isOnline = data.isOnline;
```

Comment:
Since this only really applies for private chats, consider patching this in the `PrivateChat` model, instead of the more general `Chat` one.